src/bare_metal_billing/billing.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass
+from decimal import Decimal
+
+from bare_metal_billing.models import ProjectUsage, BMUsageData
+
+
+@dataclass
+class SURates:
+    su_rates: dict[str, Decimal]
+
+
+@dataclass
+class SUInfo:
+    su_info: dict[str, dict[str, Decimal]]
+
+
+@dataclass
+class InvoiceProcessor:
+    project_invoices: list[ProjectUsage]
+    cf_info: list
+    su_info: SUInfo
+
+    def _apply_cf_info(self):
+        pass
+
+    def _apply_su_info(self):
+        pass
+
+    def process():
+        pass
+
+
+@dataclass
+class InvoiceWriter:
+    HEADERS = [
+        "Invoice Month",
+        "Project - Allocation",
+        "Project - Allocation ID",
+        "Invoice Email",
+        "Invoice Address",
+        "Institution",
+        "Institution - Specific Code",
```

Comment:
This should probably be `Institution-specific Code`, because `Institution-specific` is a compound word.

Reply:
@larsks The string `"Institution - Specific Code"` was chosen as part of our billing specification, and so it [appears](https://github.com/CCI-MOC/openshift-usage-scripts/blob/fa8e79a298d301663826c32bc4c7d2aa05ee1d61/openshift_metrics/utils.py#L132) in our [various](https://github.com/CCI-MOC/openstack-billing-from-db/blob/5b980749a8bd4d95834ace675b51d87f02fb4abb/src/openstack_billing_db/billing.py#L197) billing repo. I don't mind turning it into `Institution-specific`, but that seems like a policy question beyond me

Reply:
@larsks what you are saying is correct and makes sense but this was how were asked to generate the report and I guess we didn't notice it then. We should probably change it in a different PR because we'll need to update all the other reports (openstack, openshift, storage) as well. But before that, I want to ask @joachimweyl if this change is acceptable.

That also means that either the code will need to work with both of these column names or we'll need to re-use a specific version of our billing code if we needed to regenerate invoices from the past.

Reply:
I suggested this when we were creating the requirements and it was vetoed. Let's leave it as requested by the requirements.